libnopegl/src/backends/gl/texture_gl.c

```diff
@@ -372,14 +377,6 @@ int ngli_texture_gl_init(struct texture *s, const struct texture_params *params)
             s_priv->target == GL_TEXTURE_3D ||
             s_priv->target == GL_TEXTURE_CUBE_MAP)
             ngli_glTexParameteri(gl, s_priv->target, GL_TEXTURE_WRAP_R, wrap_r);
-
-        if (!params->width || !params->height ||
-            ((params->type == NGLI_TEXTURE_TYPE_3D ||
-              params->type == NGLI_TEXTURE_TYPE_2D_ARRAY) && !params->depth)) {
-            LOG(ERROR, "invalid texture type %dx%dx%d",
-                params->width, params->height, params->depth);
-            return NGL_ERROR_INVALID_ARG;
-        }
```

Comment:
I'm assuming the user sanity checks in tests in `node_texture.c` are enough?

Reply:
I've added some sanity checks in node_texture and node_rtt.